src/Servers/IIS/AspNetCoreModuleV2/Symbols/Microsoft.AspNetCore.ANCMSymbols.csproj

```diff
@@ -30,6 +30,9 @@
   <ItemGroup Condition=" '$(BuildIisNativeProjects)' == 'true' ">
     <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj" Platform="$(NativeVCPlatform)" />
     <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj" Platform="$(NativeVCPlatform)" />
+
+    <UpToDateCheckInput Include="$(AspNetCoreModuleV2OutOfProcessHandlerDll)" />
+    <UpToDateCheckInput Include="$(AspNetCoreModuleV2ShimDll)" />
```

Comment:
Would it help to set `$(GeneratePackageOnBuild)` to `true` and add an `@(UpToDateCheckBuilt)` item for the produced package❔

Reply:
Are you proposing that to solve the fact that this project doesn't generate an output assembly, or for some other reason?

Reply:
Yes, to solve the lack of an output assembly -- if it works 😃